examples/era5_processing_weather_data_multi_locations.py

```diff
@@ -0,0 +1,160 @@
+"""
+Read the single_location examples first.
+
+"""
+
+import json
+import os
+
+import geopandas as gpd
+import requests
+import xarray as xr
+from matplotlib import pyplot as plt
+from shapely.geometry import GeometryCollection
+from shapely.geometry import Point
+from shapely.geometry import Polygon
+from shapely.geometry import shape
+
+from feedinlib import Photovoltaic
+from feedinlib import WindPowerPlant
+from feedinlib.era5 import weather_df_from_era5
+
+
+def processing_multi_locations():
```

Comment:
I would call the function processing_multiple_locations.

Reply:
Second.